ui/tests/acceptance/console-test.js

```diff
@@ -13,13 +18,31 @@ module('Acceptance | console', function(hooks) {
   test("refresh reloads the current route's data", async function(assert) {
     await enginesPage.visit();
     let numEngines = enginesPage.rows.length;
-    await enginesPage.consoleToggle();
+    await consoleComponent.toggle();
     let now = Date.now();
     for (let num of [1, 2, 3]) {
       let inputString = `write sys/mounts/${now + num} type=kv`;
-      await enginesPage.console.runCommands(inputString);
+      await consoleComponent.runCommands(inputString);
     }
-    await enginesPage.console.runCommands('refresh');
+    await consoleComponent.runCommands('refresh');
     assert.equal(enginesPage.rows.length, numEngines + 3, 'new engines were added to the page');
   });
+
+  test('fullscreen command expands the cli panel', async function(assert) {
```

Comment:
I think these assertions are pretty solid. The only scenario which I can think of (although the assertions below may cover it) would be to test that the nav itself is still visible when the console is fullscreen.

Reply:
Yep, so in fullscreen, the nav isn't supposed to be on the page (we animate it up while we animate the console panel down).